src/base_images/javascript/Dockerfile

```diff
@@ -1,16 +1,7 @@
-# Use Node.js Alpine as base image
-FROM node:alpine
+FROM node:20-alpine
 
 WORKDIR /usr/app
 
-# Copy application files
 COPY ./ /usr/app
 
-# Install dependencies
-RUN npm install
-
-# Set environment variable
-ENV NODE_ENV=development
-
-# Run the application
-CMD ["node", "index.js"]
\ No newline at end of file
+RUN npm install --production
```

Comment:
Is the production vs development flags in the diff no longer relevant when spinning this up locally vs in our cloud infra?